### ISSUE TITLE
Update Openlitespeed to BUILD built: Mon Oct 17 21:33:28 UTC 2022

### ISSUE THREAD

**Issue by jordantrizz** [litespeedtech/cyberpanel#999]
Provisioned CyberPanel instances before October 17th are running an older version of Openlitespeed.

`LiteSpeed/1.7.16 Open (BUILD built: Fri May 13 19:00:23 UTC 2022)`

The newest build is

`LiteSpeed/1.7.16 Open (BUILD built: Thu Nov 17 16:18:46 UTC 2022)`

Granted `apt-get update;apt-get install --only-upgrade` will install the new build.

Release a new version of CyberPanel that updates the openlitespeed package and informs users an update is available would be good industry practice, especially since the build before the newest includes security fixes that should be applied.

**Comment by usmannasir** [comment]
Where did you get CyberPanel from? If you install CyberPanel from our installation guide it always gets you latest version.

**Comment by jordantrizz** [comment]
Correct, but what about if you installed CyberPanel before the latest Openlitespeed release? When does CyberPanel update Openlitespeed? You didn't send out a release.

**Comment by master3395** [comment]
Thanks for the report. This issue is older and may not apply to current releases.

Please confirm:
1) CyberPanel version + build (please retest on the latest public release **2.4.9**)
2) OS name and version (whatever you run, e.g. `cat /etc/os-release`)
3) Exact steps still failing on a current install
4) Relevant logs (no passwords)

Note: the latest public release is **2.4.9**. Development branches are not recommended for reproducing bugs.

If we do not hear back within 30 days, this issue may be closed as stale. You can reopen with fresh details anytime.
